client/js/views/Header.js

```diff
@@ -12,11 +12,8 @@ module.exports = Object.create( Object.assign( {}, require('./__proto__'), {
 
     name: 'Header',
     onNavListClick(event) {
-        this.emit('navigate', `/${ event.target.textContent.split(' ').join('') }`)
-        var headers = event.target.parentElement.children
-        for (var i = 0; i < headers.length; i ++) {
-            headers[i].classList.remove('selected')
-        }
+        this.emit('navigate', `/${ event.target.textContent.replace(/\s+/g, '') }`)
```

Comment:
Minor optimization nit: generally, RegExp is slow, so it would be best to using strings in place of them whenever possible.  Sorry for steering you in the RegExp direction with my `capitalizeWords` method above ( I should have used strings )

Reply:
So instead of using `event.target.textContent.replace(/\s+/g, '')` would it be better to use `event.target.textContent.replace(' ', '')` to remove the whitespaces?